modules/grafana-lgtm/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/testcontainers/testcontainers-go/modules/grafanalgtm
+module github.com/testcontainers/testcontainers-go/modules/grafana-lgtm
```

Comment:
I need to verify the implications on changing the module name of an already published module (appeared in v0.33.0)

Reply:
never ideal but its is pre v1.0 so this is the time to do it and guessing it wasn't working?